### ISSUE TITLE
Could Levenshtein transformer be trained normal?loss is always rising...

### ISSUE THREAD

**Issue by jiezhangGt** [pytorch/fairseq#2188]
Hi, I'm trying to train a LevT model with Encoder layers=12, Decoder layers=3, but the training loss is always rising. 

Here is my training set:

```bash
#!/usr/bin/bash

BIN=./fairseq-0.9.0/fairseq-0.9.0
BASEDIR=.


train_data=$BASEDIR/data/train_data_distill
saveDir=$BASEDIR/checkpoints_distill_e12d3
ARCH='levenshtein_transformer_vaswani_wmt_en_de_e12d3'
TASK='translation_lev'

ChunkData=$(ls $train_data)
ALLDATA=""
for filechunk in ${ChunkData};do
	if [ ! -d $train_data/$filechunk ];then
		continue
	fi
	ALLDATA=${ALLDATA}:$train_data/${filechunk}
done

if [ ${#ALLDATA} -eq 0 ]; then
	ALLDATA=$train_data
else
	ALLDATA=${ALLDATA:1:${#ALLDATA}}
fi

echo $ALLDATA


# export CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6
python -u $BIN/train.py $ALLDATA --save-dir $saveDir --log-format 'simple' --ddp-backend 'no_c10d' \
					--source-lang 'en' --target-lang 'zh' --left-pad-source False --left-pad-target False \
					--max-source-positions 384 --max-target-positions 384 \
					--arch $ARCH --task $TASK \
					--noise 'random_delete' \
					--apply-bert-init \
					--max-tokens 6000 --max-sentences 2000 --max-epoch 1000 --max-update 1000000 --save-interval-updates 2000 --save-interval 10000 --log-interval 50 --update-freq 10 \
					--lr-scheduler 'inverse_sqrt' --learning-rate 0.0004 --min-lr 1e-10 \
					--warmup-updates 10000 --warmup-init-lr 1e-7 \
					--criterion 'nat_loss' --label-smoothing 0.1 --weight-decay 0.01 \
					--optimizer 'adam' --adam-betas '(0.9, 0.98)' --fp16

```

![微信图片_20200527151032](https://user-images.githubusercontent.com/42362818/82988777-441d5680-a02c-11ea-8b31-d1833b48541b.png)

**Comment by myleott** [comment]
Any ideas @MultiPath?

**Comment by jiezhangGt** [comment]
> Any ideas @MultiPath?
Can someone help me out?

**Comment by MultiPath** [comment]
Hi, @jiezhangGt  sorry for not noticing this for a long time. Is the problem solved?
